docker/Dockerfile.build-linux

```diff
@@ -0,0 +1,10 @@
+FROM ruby:2.4-slim-stretch
```

Comment:
FYI, I see Ruby 2.5.x is stable now if you were interested in moving up.

Reply:
👍 , but as a separate PR.  Upgrading to a new ruby version should also update CI etc..